src/overlays/actors/ovl_En_Md/z_en_md.c

```diff
@@ -9,7 +9,43 @@ void EnMd_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnMd_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnMd_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_80AAA250(EnMd* this);
```

Comment:
We generally want to avoid forward declaring functions that don't need to be, especially functions like OverrideLimbDraw.